rust/src/db.rs

```diff
@@ -196,3 +211,196 @@ pub async fn insert_ignore_txid(txid: Txid, maker_amount: i64) -> Result<()> {
 
     Ok(())
 }
+
+pub async fn insert_payment(payment: &PaymentInfo) -> Result<()> {
+    let mut conn = acquire().await?;
+
+    let PaymentInfo {
+        hash,
+        preimage,
+        secret,
+        status,
+        amt_msat,
+        created_timestamp,
+        updated_timestamp,
+    } = payment.clone();
+
+    let query_result = sqlx::query(
```

Comment:
🔧 Shouldn't we use the `query!` macro to get some compile-time safety?

Reply:
Note: In itchysats we used the `query!` macro for `select` statements, but not for inserts. Not sure what was the reason though.

Reply:
I just followed the other insert statement, I will see whether both can be `query!` instead. thanks for spotting 🙏

Reply:
I couldn't make this work easily, I left it as all the other insert statements as it's not the priority right now.